experiments/long_runs/data_paper_plots.jl

```diff
@@ -0,0 +1,428 @@
+            return sim_var_lwu
+        end
+
+    # Read in SWU
+    sim_var_dict["swu"] =
+        () -> begin
+            sim_var_swu = get(simdir, short_name = "swu") # units (W/m²)
+            sim_var_swu.attributes["long_name"] = "Upward shortwave radiation"
+            sim_var_swu.attributes["units"] = "W m⁻²"
+            return sim_var_swu
+        end
+
+    # Read in LWD
+    sim_var_dict["lwd"] =
+        () -> begin
+            sim_var_lwd = get(simdir, short_name = "lwd") # units (W/m²)
+            sim_var_lwd.attributes["long_name"] = "Downward longwave radiation"
+            sim_var_lwd.attributes["units"] = "W m⁻²"
+            return sim_var_lwd
+        end
+
+    # Read in SWD
+    sim_var_dict["swd"] =
+        () -> begin
+            sim_var_swd = get(simdir, short_name = "swd") # units (W/m²)
+            sim_var_swd.attributes["long_name"] = "Downward shortwave radiation"
+            sim_var_swd.attributes["units"] = "W m⁻²"
+            return sim_var_swd
+        end
+
+    # Compute LWN = LWD - LWU
+    sim_var_dict["lwn"] =
+        () -> begin
+            lwd = sim_var_dict["lwd"]()
+            lwu = sim_var_dict["lwu"]()
+            lwn = lwd - lwu
+            lwn.attributes["long_name"] = "Net longwave radiation"
+            lwn.attributes["units"] = "W m⁻²"
+            lwn.attributes["start_date"] = lwu.attributes["start_date"]
+            return lwn
+        end
+
+    # Compute SWN = SWD - SWU
+    sim_var_dict["swn"] =
+        () -> begin
+            swd = sim_var_dict["swd"]()
+            swu = sim_var_dict["swu"]()
+            swn = swd - swu
+            swn.attributes["long_name"] = "Net shortwave radiation"
+            swn.attributes["units"] = "W m⁻²"
+            swn.attributes["start_date"] = swu.attributes["start_date"]
+            return swn
+        end
+
+    return sim_var_dict
+end
+
+"""
+    get_obs_var_dict()
+
+Retrieve the following variables from ERA5 observational data:
+- lhf
+- shf
+- lwu
+- swu
+- lwd
+- swd
+- lwn (computed as lwd - lwu)
+- swn (computed as swd - swu)
+
+Note: `get_obs_var_dict` in ext/land_sim_vis/leaderboard/data_sources.jl uses
+the artifact `era5_monthly_averages_single_level_path`, which does not include lwd and swd.
+
+This function does the same, but also gets lwd and swd from `era5_land_forcing_data2008_folder_path`.
+"""
+function get_obs_var_dict(; is_local = false)
+    # contains monthly mslhf, msshf, msuwlwrf, msuwswrf
+    era5_data_path = joinpath(
+        ClimaLand.Artifacts.era5_monthly_averages_single_level_path(),
+        "era5_monthly_averages_surface_single_level_197901-202410.nc",
+    )
+    # contains hourly msdwlwrf, msdwswrf
+    if is_local
+        era5_land_forcing_data_path = ClimaLand.Artifacts.era5_land_forcing_data2008_lowres_path()
+    else
+        # TODO don't hard code these
+        era5_land_forcing_data_path = ClimaLand.Artifacts.find_era5_year_paths(
+            DateTime("2008-03-01"), # start_date
```

Comment:
possible bug here?

Reply:
This is fixed in my current version. I'll push now